### PR TITLE
Add Color.toCssHexString and unit tests

### PR DESCRIPTION
**Pull request by ErixenCruz**
Added `Color.toCssHexString` for getting the CSS hex string equivalent for a color.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @ErixenCruz! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/CesiumGS/cesium/blob/color-CssHexString/CONTRIBUTORS.md) file!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/Core/Color.js [hunk 1]
```diff
@@ -632,6 +632,27 @@ Color.prototype.toCssColorString = function () {
   return "rgba(" + red + "," + green + "," + blue + "," + this.alpha + ")";
 };
 
+/**
+ * Creates a string containing CSS hex string color value for this color.
```
Comment (hpinkos) [inline Source/Core/Color.js:636]: There should be some kind of note that this ignores the alpha value of the color.

**Comment by mramato** [comment]
What's the use case for this?  `toCssColorString` can be used in CSS anywhere `toCssHexString` could also be used, right?  And it supports alpha.

**Comment by mramato** [comment]
Also, hex strings with alpha work everywhere but IE11, so I'm not sure why we would leave alpha off: https://caniuse.com/#feat=css-rrggbbaa

**Comment by hpinkos** [comment]
I had a use case for this.  There was a 3rd party component i wanted to send a color value to that only accepted hex strings

**Comment by hpinkos** [comment]
+1 for using `RRGGBBAA` if alpha < 1 though, instead of just ignoring it

**Comment by ErixenCruz** [comment]
@hpinkos Updated with your feedback.

File: CHANGES.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ##### Breaking Changes :mega:
 
+- Added `Color.toCssHexString` for getting the CSS hex string equivalent for a color.
```
Comment (hpinkos) [inline CHANGES.md:7]: This should go under the `Additions` section instead of the `Breaking Changes` section

**Comment by hpinkos** [comment]
@ErixenCruz Be sure to add yourself to https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTORS.md under the CesiumGS section!

**Comment by ErixenCruz** [comment]
I added myself in https://github.com/CesiumGS/cesium/pull/8983. Should I add myself here, too? @hpinkos

**Comment by hpinkos** [comment]
@ErixenCruz yes please!  I think this PR will be ready to be merged first, and we should add your name to the contributors list when this goes in

**Comment by ErixenCruz** [comment]
@hpinkos Pushed commit with your feedback. Thanks.

File: CHANGES.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 ##### Additions :tada:
 
+- Added `Color.toCssHexString` for getting the CSS hex string equivalent for a color.
```
Comment (lilleyse) [inline CHANGES.md:11]: Make sure to include a link to the PR in the `CHANGES.md` entry

```suggestion
- Added `Color.toCssHexString` for getting the CSS hex string equivalent for a color. [#8987](https://github.com/CesiumGS/cesium/pull/8987)
```

**Comment by ErixenCruz** [comment]
@lilleyse Updated.

**Comment by lilleyse** [comment]
@ErixenCruz thanks! Looks good.

**Comment by nmschulte** [comment]
This is very useful, thank you!

**Pull request merged**